openmindat/localities.py

```diff
@@ -191,9 +193,12 @@ def expand(self, EXPAND_FIELDS):
         if invalid_options:
             raise ValueError(f"Invalid EXPAND_FIELDS: {', '.join(invalid_options)}\nEXPAND_FIELDS must be one or more of the following: {', '.join(valid_options)}")
 
-        expand_fields = EXPAND_FIELDS
+                #Changes * to "*" and from a list to a string since this is what the param will read without failing.
+        expand_fields = [w.replace('*', '"*"') for w in expand_fields]
```

Comment:
I am wondering if the initial user input is a `"*"` and your code replaces it with `""*""`. Will the API server still recognize and process it? Again, do not `raise Value Error` for the "invalid" inputs, but printing it as a soft warning would be better.